src/Microsoft.PowerShell.Commands.Diagnostics/project.json

```diff
@@ -10,6 +10,30 @@
         "warningsAsErrors": true
     },
 
+    "configurations": {
+        "Linux": {
+            "buildOptions": {
+                "define": [ "UNIX" ],
+                "debugType": "portable"
+            }
+        },
+        "Debug": {
+                "buildOptions": {
+                    "debugType": "portable"
+                }
```

Comment:
@daxian-dbw Fixed. Sorry missed the buildOptions. Had only fixe the closing braces. Now it is fixed.